src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -635,7 +635,7 @@ private static function mergeQueryString(?string $queryString, array $queryArray
     private static function getProxy(?string $proxy, array $url, ?string $noProxy): ?array
     {
         if (null === $proxy) {
-            // Ignore HTTP_PROXY except on the CLI to work around httpoxy set of vulnerabilities
+            // Ignore HTTP_PROXY except on the CLI to work around httproxy set of vulnerabilities
```

Comment:
https://httpoxy.org/ this probably was intended

Reply:
Thanks. reverted.